doc/doc-cheat-sheet.md

```diff
@@ -0,0 +1,545 @@
+---
```

Comment:
This file is currently not included in the navigation, but it can be accessed directly by URL.